src/load/azext_load/_help.py

```diff
@@ -9,3 +9,8 @@
 # pylint: disable=too-many-lines
 
 from knack.help_files import helps  # pylint: disable=unused-import
+from azext_load.data_plane.help import helps as data_common_helps
+from azext_load.data_plane.load_test.help import helps as data_load_test_helps
```

Comment:
load_test_run.help ?

Reply:
Yeah a bit of test run is remaining. Will add those

Reply:
Have added test_run_help as well